### PR TITLE
Add integration tests for 'cs setup'

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

File: modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    test("setup") {
+      TestUtil.withTempDir { tempDir =>
+        val result = os.proc(
+          launcher,
+          "setup",
+          "--yes",
+          "--install-dir",
+          tempDir.getAbsolutePath
+        ).call(cwd = os.Path(tempDir, os.pwd))
+        assert(result.exitCode == 0)
```
Comment (julienrf) [inline modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala:20]: Is there anything stronger I could assert?

**Review by alexarchambault — COMMENTED** [review]
`--user-home` should allow to make more reproducible, without messing with the global env.

File: modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala (hunk 1 above)
Comment (julienrf) [reply modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala:24]: Done in a4fe96093

File: modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          "--yes",
+          "--user-home",
+          homeDir.toString,
+          "--install-dir",
+          installDir.toString
+        ).call()
+        assert(result.exitCode == 0)
+
+        for (app <- List("scala", "sbt", "cs"))
+          assert(os.exists(installDir / app))
```
Comment (julienrf) [inline modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala:27]: That failed on Windows: https://github.com/coursier/coursier/actions/runs/3940975124/jobs/6742770477

Did I make a mistake in the way I build the path?

Comment (alexarchambault) [reply modules/cli-tests/src/main/scala/coursier/clitests/SetupTests.scala.scala:27]: I think a `.bat` extension is needed on Windows.

**Review by alexarchambault — APPROVED** [review]
👍

**Pull request merged**